src/main/java/duke/helper/Ui.java

```diff
@@ -0,0 +1,86 @@
+package duke.helper;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.task.Task;
+
+/**
+ * Ui class that handles all the interactions with the users
+ */
+public class Ui {
+    private Scanner scanner;
+
+    public Ui() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    public static String showErrorMsg(String desc) {
+        return String.format("%s commands need to be followed by an integer!\n", desc.split(" ")[0]);
```

Comment:
These can be abstracted into Exceptions, and all the Exceptions may be passed into a single UI method that is responsible for displaying Exceptions. You can then unit test that the exceptions are output correctly to the console.